docs/UsersGuide/source/BuildSRW.rst

```diff
@@ -0,0 +1,495 @@
+
+Run the executable that pulls in SRW App components from external repositories:
+
+.. code-block:: console
+
+   cd $SRW
+   ./manage_externals/checkout_externals
+
+The script should output dialogue indicating that it is retrieving different code repositories. It may take several minutes to download these repositories.
+
+To see more options for the ``checkout_externals`` script, users can run ``./manage_externals/checkout_externals -h``. For example:
+
+   * ``-S``: Outputs the status of the repositories managed by ``checkout_externals``. By default only summary information is provided. Use with the ``-v`` (verbose) option to see details.
+   * ``-x [EXCLUDE [EXCLUDE ...]]``: allows users to exclude components when checking out externals. 
+   * ``-o``: By default only the required externals are checked out. This flag will also checkout the optional externals.
+
+Generally, users will not need to use the options and can simply run the script, but the options are available for those who are curious. 
+
+.. _BuildExecutables:
+
+Set Up the Environment and Build the Executables
+===================================================
+
+.. _DevBuild:
+
+``devbuild.sh`` Approach
+-----------------------------
+
+On Level 1 systems for which a modulefile is provided under the ``modulefiles`` directory, users can build the SRW App binaries with the following command:
+
+.. code-block:: console
+
+   ./devbuild.sh --platform=<machine_name>
+
+where ``<machine_name>`` is replaced with the name of the platform the user is working on. Valid values are: ``cheyenne`` | ``gaea`` | ``hera`` | ``jet`` | ``linux`` | ``macos`` | ``noaacloud`` | ``odin`` | ``orion`` | ``singularity`` | ``wcoss2``
+
+.. note::
+   Although build modulefiles exist for generic Linux and MacOS machines, users will need to alter these according to the instructions in Sections :numref:`%s <CMakeApproach>` & :numref:`%s <MacDetails>`. Users on these systems may have more success building the SRW App with the :ref:`CMake Approach <CMakeApproach>` instead. 
+
+If compiler auto-detection fails for some reason, specify it using the ``--compiler`` argument. For example:
+
+.. code-block:: console
+
+   ./devbuild.sh --platform=hera --compiler=intel
+
+where valid values are ``intel`` or ``gnu``.
+
+The last line of the console output should be ``[100%] Built target ufs-weather-model``, indicating that the UFS Weather Model executable has been built successfully. 
+
+If users want to build the optional ``GSI`` and ``rrfs_utl`` components for :term:`RRFS`, they can pass the ``gsi`` and ``rrfs_utils`` arguments to ``devbuild.sh`` (NOTE: These components are not currently available for use at runtime). For example:
+
+.. COMMENT: What does the runtime note mean?
```

Comment:
@gspetro-NOAA I believe that the runtime note is referring to the fact that, even if the GSI and rrfs_utl components are built, the executables will not be used in any of the current WE2E tests.  When RRFS cycling tests are added, then these executables will be used.  Maybe @christinaholtNOAA or @christopherwharrop can weigh in on this (I want to make sure this is correct)?

Reply:
@gspetro-NOAA : @MichaelLueken is correct.  It means that GSI and rrfs_utl are not currently used in any workflows.  In other words, you can build GSI and rrfs_utl, but you can't run them.  This was done because it was a huge amount of work just to get those components added to the build.  And even more work to get RRFS DA workflows enabled in the authoritative repository.  We haven't got to the work for adding the RRFS DA capabilities yet.